docs/src/examples/components/Chat/Types/ChatExample.tsx

```diff
@@ -8,44 +8,60 @@ const janeAvatar = {
 
 const ChatExample = () => (
   <Chat>
-    <Chat.Item>
-      <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
-    </Chat.Item>
-    <Chat.Item>
-      <Chat.Message
-        content="Hi"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    </Chat.Item>
-    <Chat.Item>
-      <Chat.Message
-        content="Would you like to grab a lunch?"
-        author="John Doe"
-        timestamp="Yesterday, 10:16 PM"
-        mine
-      />
-    </Chat.Item>
-    <Chat.Item>
-      <Chat.Message
-        content="Sure! Let's try the new place downtown"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    </Chat.Item>
+    <Chat.Item
+      content={
+        <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
+      }
+    />
+    <Chat.Item
+      gutterPosition="start"
+      gutter={
+        <Chat.Gutter>
+          <Avatar {...janeAvatar} />
```

Comment:
Why we even need to use Chat.Gutter here?

Reply:
see https://github.com/stardust-ui/react/pull/556#discussion_r240997497
even though 1st point is not that strong